src/js/components/Menu/__tests__/Menu-test.js

```diff
@@ -522,4 +535,16 @@ describe('Menu', () => {
     );
     expect(component.toJSON()).toMatchSnapshot();
   });
+
+  test('custom theme with default button', () => {
```

Comment:
Would it be helpful to build a snapshot test to cover the scenario where `button.default` is defined and the Menu has children? This would help ensure that the logic of rendering children if present continues to be applied.

Reply:
Yes, I think you're right. We should have that as a baseline.

Reply:
Added this.